htdocs/holiday/card.php

```diff
@@ -449,10 +449,9 @@
 				$subject = $societeName." - ".$langs->transnoentitiesnoconv("HolidaysToValidate");
 
 				// Content
-				$message = $langs->transnoentitiesnoconv("Hello")." ".$destinataire->firstname.",\n";
-				$message .= "\n";
+				$message = "<p>".$langs->transnoentitiesnoconv("Hello")." ".$destinataire->firstname.",</p>\n";
```

Comment:
If you include <p> tags, it means the content will be html, so new line should be <br> and not \n.
Are you sure the rendering of email is correct with this change ?